ariadne/cardano/src/Ariadne/Wallet/Backend/KeyStorage.hs

```diff
@@ -425,6 +433,32 @@ renameSelection pwl WalletFace{..} walletSelRef name = do
 
   walletRefreshState
 
+
+-- | This function removes unknown keys from key file on startup, if user confirms that.
+checkUnknownKeys
+  :: PassiveWalletLayer IO
+  -> (ConfirmationType -> IO Bool)  -- ^ Asks user permission to delete unknown keys
+  -> IO ()
+checkUnknownKeys pwl confirmation = do
+  unknownRootIds <- pwlGetUnknownKeys pwl
+  unless (null unknownRootIds) $ do
+    let unknownRootIdsText =
+          T.dropEnd 1 . unlines $ T.drop 10 . toText . sformat build <$> unknownRootIds
```

Comment:
`sformat build` returns `Text`, so `toText` is most likely redundant (unless I'm missing something).

Reply:
You are right, just forget to remove. Fixed.